admin-dev/themes/new-theme/js/product-page/combination.js

```diff
@@ -1,57 +1,59 @@
+var $ = window.$;
+
 export default function() {
   $(document).ready(function() {
-    let $jsCombinationsList = $('.js-combinations-list');
+    const $jsCombinationsList = $('.js-combinations-list');
     // If we are not on the product page, return
     if (0 === $jsCombinationsList.length) {
         return;
     }
-    let idsProductAttribute = $jsCombinationsList.data('ids-product-attribute').toString().split(',');
-    let idsCount = idsProductAttribute.length;
+
+    const idsProductAttribute = $jsCombinationsList.data('ids-product-attribute').toString().split(',');
+    const refreshImagesUrl = $jsCombinationsList
+      .attr('data-action-refresh-images')
+      .replace(/product-form-images\/\d+/, 'product-form-images/' + $jsCombinationsList.data('id-product'));
+    const idsCount = idsProductAttribute.length;
+    const step = 50;
     let currentCount = 0;
-    let step = 50;
 
-    let refreshImagesUrl = $jsCombinationsList.attr('data-action-refresh-images').replace(/product-form-images\/\d+/, 'product-form-images/' + $jsCombinationsList.data('id-product'));
 
-    $.get(refreshImagesUrl)
-      .then(function(response) {
-        if (idsProductAttribute[0] != '') {
-          getCombinations(response);
-        }
-      });
+    $.get(refreshImagesUrl).then(function(response) {
+      if (idsProductAttribute.length !== 0) {
```

Comment:
You could you directly `idsCount`

However, by looking at the previous code I wonder if you can get an empty value in the first row.

Reply:
Done, thanks ;)